mutate-my-strings/solution.js

```diff
@@ -0,0 +1,13 @@
+function mutateMyStrings(str1, str2) {
+  let solution = '';
+
+  for (let i = 0; i < str1.length; i++) {
+    if (str1[i] === str2[i]) {

```

Comment:
It'd be cleaner to simply do an 'if not' comparison, then execute the code that is currently sitting in the else statement.

Anytime you want to do a `continue` without any other code in an if/else statement, this is usually a good indicator that it is not necessary.